draft-ietf-jsonpath-base.md

```diff
@@ -672,16 +681,14 @@ unescaped           = %x20-21 /                         ; s. RFC 8259
                       %x28-5B /                         ; omit '
                       %x5D-10FFFF                       ; omit \
 
-escapable           = ( %x62 / %x66 / %x6E / %x72 / %x74 / ; \b \f \n \r \t
-                          ; b /         ;  BS backspace U+0008
-                          ; t /         ;  HT horizontal tab U+0009
-                          ; n /         ;  LF line feed U+000A
-                          ; f /         ;  FF form feed U+000C
-                          ; r /         ;  CR carriage return U+000D
-                          "/" /          ;  /  slash (solidus)
-                          "\" /          ;  \  backslash (reverse solidus)
-                          (%x75 hexchar) ;  uXXXX      U+XXXX
-                      )
+escapable           = %x62 /         ; b  BS backspace U+0008
+                      %x66 /         ; f  FF form feed U+000C
+                      %x6E /         ; n  LF line feed U+000A
+                      %x72 /         ; r  CR carriage return U+000D
+                      %x74 /         ; t  HT horizontal tab U+0009
+                      %x47 /         ; /  slash (solidus)
```

Comment:
We do not use "b" because that would be case insensitive in ABNF.
We currently use %x62, we could use %s"b" instead.
But we never need to use %x47, as "/" is unambiguous.

Reply:
ok ,,, got it.